documentation/configuration.md

```diff
@@ -153,7 +153,12 @@ Example:
 
 ### fetcher :: atom()
 
-default: Crawly.Fetchers.HTTPoisonFetcher 
+default: Crawly.Fetchers.HTTPoisonFetcher
 
-Allows to specify a custom HTTP client which will be performing request to the crawl
-target.
\ No newline at end of file
+Allows to specify a custom HTTP client which will be performing request to the crawler target.
+
+### port :: pos_integer()
+
+default: 4001
+
+Allows to specify a custom port to start the application, that is important when running more than one crawler (or other applications) in a single machine.
```

Comment:
Just to double-check. It's possible to run multiple Crawlers. You can't start multiple instances of Crawly. And in general, it's possible to run as many individual Crawlers as you want. could you please update the text accordingly, and I will gladly merge it!

Reply:
Ok, it really don't make sense.
What do you think of:
```
Allows to specify a custom port to start the application, that is important when running more than one applications that can use the same port  in a single machine.
```

Reply:
Sounds good

Reply:
I changed it a little bit, I it is clearer in this way